tests/Kernel/Paragraphs/ContentRowTest.php

```diff
@@ -17,6 +17,7 @@ class ContentRowTest extends ParagraphsTestBase {
    * Tests the rendering of the paragraph type.
    *
    * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+   * @todo Update (restore) after porting to ECL2 other components.
```

Comment:
Can you add the number of the ticket?

Reply:
Thanks. I have added mentioning of related tickets.